src/main/java/duke/tasks/TaskList.java

```diff
@@ -0,0 +1,79 @@
+package duke.tasks;
+
+import duke.storage.Storage;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class TaskList {
+    private static final ArrayList<Task> taskList = new ArrayList<>();
+
+    static String countTasks() {
+        String pluralSuffix = taskList.size() == 1 ? "" : "s";
+        return String.format("%s task%s", taskList.size(), pluralSuffix);
+    }
+
+    static String add(Task task) {
+        taskList.add(task);
+
+        return String.format("Great, have added the following task for you:\n%s\nNow you have %s.", task, countTasks());
+    }
+
+    static Task get(int index) {
+        return taskList.get(index - 1);
+    }
```

Comment:
I think `index` should already have been `entryNumber - 1`, this _might_ cause some confusion but it's really just renaming it as:
```suggestion
    static Task get(int entryNumber) {
        return taskList.get(entryNumber - 1);
    }
```